frontend/public/src/components/ReceiptPageDetailCard.js

```diff
@@ -0,0 +1,225 @@
+                          : null}
+                      </dd>
+                    </dl>
+                  </div>
+                ) : orderReceipt.transactions.payment_method === "paypal" ? (
+                  <div>
+                    <dl>
+                      {orderReceipt.transactions.bill_to_email ? (
+                        <dl>
+                          <dt>Email:</dt>
+                          <dd>{orderReceipt.transactions.bill_to_email}</dd>
+                        </dl>
+                      ) : null}
+                    </dl>
+                    <dl>
+                      <dt>Payment Method:</dt>
+                      <dd id="paymentMethod">Paypal</dd>
+                    </dl>
+                  </div>
+                ) : null}
+              <div>
+                <dl>
+                  <dt>Discount Code:</dt>
+                  <dd>{discounts[0].discount_code}</dd>
+                </dl>
+                <dl>
+                  <dt>Address:</dt>
+                  <dd>
+                    {orderReceipt.street_address.line.map(_line => (
+                      <div
+                        className="value low-line-height"
+                        key={_line}
+                        id={_line}
+                      >
+                        {_line}
+                      </div>
+                    ))}
+                    <div className="value low-line-height" id="purchaserState">
+                      {orderReceipt.street_address.city}, {stateCode}{" "}
+                      {orderReceipt.street_address.postal_code}
+                    </div>
+                    <div
+                      className="value low-line-height"
+                      id="purchaserCountry"
+                    >
+                      {orderReceipt.street_address.country}
+                    </div>
+                  </dd>
+                </dl>
+              </div>
+            </div>
+          ) : null}
+        </div>
+        <div className="receipt-table-holder">
+          <h3>Product Description</h3>
+          <table className="receipt-table">
+            <thead>
+              <tr>
+                <th>Product Description</th>
+                <th>Quantity</th>
+                <th>Unit Price</th>
+                <th>Discount</th>
+                <th>Total Paid</th>
+              </tr>
+            </thead>
+            <tbody>
+              {orderReceipt.lines.map(line => {
+                const startDate = parseDateString(
+                  line.product.purchasable_object.start_date
+                )
+                const endDate = parseDateString(
+                  line.product.purchasable_object.end_date
+                )
+                return (
+                  <tr key={line.product.purchasable_object.readable_id}>
+                    <td>
+                      <div>
+                        {line.product.purchasable_object.title} <br />
```

Comment:
Can you update this to get the course's readable_id rather than the one for the course run? Also, can you add the `course_number` to the beginning of the title (if it's a course that's been purchased)? That will make it match the email receipt code. (Take a look at the `TransactionLineSerializer` to see how we're doing it there.)

Reply:
@jkachel If I am not wrong, It should not be CourseRun instead of a course here ?

Reply:
@jkachel If I am not wrong, It should not be CourseRun instead of a course here ?

Reply:
You mean that `line.product.purchasable_object.course.readable_id` but it can be a program.

Reply:
@jkachel Do you mean the change set as in [commit here](https://github.com/mitodl/mitxonline/pull/578/commits/4e02dd6f4633213382ae113a4edc1bfde1611fbe) ?

Reply:
<img width="1102" alt="Screenshot 2022-05-27 at 3 58 44 PM" src="https://user-images.githubusercontent.com/7334669/170686935-824266a0-5c97-43b2-bb0f-80f8909c8368.png">


Showing like this one.

Reply:
Yup - tested with an order locally and looks good to me.